design/ceph/object/store.md

```diff
@@ -33,6 +33,10 @@ spec:
     port: 80
     securePort: 443
     instances: 3
+  # radosNamespaces:
```

Comment:
Let's remove this example. We can just focus on the rados namespaces example in the later section of the doc.